.github/workflows/pull-request.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         os: [ubuntu-latest]
     env:
       CI: 1
-      NPM_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN }}
+      NPM_TOKEN: ${{ secrets.NPM_RO_TOKEN }}
```

Comment:
is `RO` correct? or a typo?

Reply:
its correct, i made a global secret for access to `NPMJS` that is scoped to `read-only` and just dropped it in as a global secret with repo restrictions vs attaching it to every repo that needed to consume it directly.

Reply:
ahh RO = ReadOnly - that wasn't clear :)